cmake/kokkos_enable_options.cmake

```diff
@@ -57,6 +57,7 @@ KOKKOS_ENABLE_OPTION(TUNING               OFF "Whether to create bindings for tu
 KOKKOS_ENABLE_OPTION(AGGRESSIVE_VECTORIZATION OFF "Whether to aggressively vectorize loops")
 KOKKOS_ENABLE_OPTION(LAUNCH_COMPILER      ON  "Whether to potentially use the launch compiler")
 KOKKOS_ENABLE_OPTION(COMPILE_AS_CMAKE_LANGUAGE OFF "Whether to use native cmake language support")
+KOKKOS_ENABLE_OPTION(EXPERIMENTAL_DEFAULT_DYNAMIC_VIEWHOOKS OFF "Whether to use the experimental dynamic view hooks by default for each view")
```

Comment:
Why is it not `VIEWHOOKS -> VIEW_HOOKS`?

Reply:
good point, fixed in new commits

Reply:
accidentally re-added when rebasing